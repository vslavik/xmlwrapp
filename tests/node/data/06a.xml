<root><chapter><title>Title</title></chapter></root>
