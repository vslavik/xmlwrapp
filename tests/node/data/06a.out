type_element
    type_element
        type_element
            type_text
